Dockerfile

```diff
@@ -1,39 +1,65 @@
 # Base layer
 FROM ubuntu:22.04 AS baseline
+
 RUN apt-get update && apt-get upgrade -y --no-install-recommends \
-  && apt-get install -y python3 python3-dev python3-pip curl unzip gnupg --no-install-recommends \
+  && apt-get install -y \
+      python3 python3-dev python3-pip \
+      curl unzip gnupg lsb-release ca-certificates software-properties-common \
```

Comment:
I understand the extra lines, but wondering about lsb-release, ca-certificates, and software-properties-common